packages/flutter_tools/lib/src/windows/install_manifest.dart

```diff
@@ -51,5 +51,5 @@ Future<void> createManifest({
   outputs.add(project.ephemeralDirectory.childFile('icudtl.dat'));
   project.ephemeralDirectory.childFile('install_manifest')
     ..createSync(recursive: true)
-    ..writeAsStringSync(outputs.map((File file) => file.absolute.path).join('\n'));
+    ..writeAsStringSync(outputs.map((File file) => file.absolute.uri.toString().split('file:///')[1]).join('\n'));
```

Comment:
I think you could just do `uri.path` here.

Reply:
done -ish 👍